smoke_tests/tests/test_running_functions.py

```diff
@@ -96,7 +96,13 @@ def test_executor(compute_client, endpoint, tutorial_function_id):
     num_tasks = 10
     submit_count = 2  # we've had at least one bug that prevented executor re-use
 
-    with Executor(endpoint_id=endpoint, client=compute_client) as gce:
+    # use client on newer versions and funcx_client on older
+    try:
+        gce = Executor(endpoint_id=endpoint, client=compute_client)
+    except TypeError:
+        gce = Executor(endpoint_id=endpoint, funcx_client=compute_client)
+
+    with gce:
```

Comment:
What about doing this with an SDK version check instead?  That then gives us a clue in N months/years when we can remove that backward-supporting branch.

Reply:
Considered that, but we haven't released the offending change yet, so we don't have a good number to compare against

Reply:
Ah; that is a problem.  Okay, no quick fix comes to mind.  We'll just "have to remember" to go fix this.